static/js/components/dashboard/CoursePrice.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import React from 'react';
+
+import type { Course } from '../../flow/programTypes';
+import {
+  STATUS_ENROLLED_NOT_VERIFIED,
+  STATUS_OFFERED_NOT_ENROLLED,
+} from '../../constants';
+import { formatPrice } from '../../util/util';
+
+export default class CoursePrice extends React.Component {
+  props: {
+    course: Course,
+  };
+
+  render() {
+    const { course } = this.props;
+    let firstRun = {};
+    let price = null;
+    let enrollmentStatus = null;
+
+    if (course.runs.length > 0) {
+      firstRun = course.runs[0];
+      if (course.status === STATUS_OFFERED_NOT_ENROLLED || course.status === STATUS_ENROLLED_NOT_VERIFIED) {
+        if (firstRun.price) {
+          price = <span className="course-price-display">{formatPrice(firstRun.price)}</span>;
+        }
+        enrollmentStatus = <span className="course-enrollment-description">Enrollment open</span>;
```

Comment:
This is out of scope for this PR, but the goal is to support all these cases in depicted in https://cloud.githubusercontent.com/assets/430126/18178388/1caecd22-704c-11e6-9328-1c7fab19b421.png

George is correct that the enrollmentStatus string depends on the date. It should probably be handled separately from the price display logic.